### PR TITLE
Make clone button work & other minor fixes

### PR DESCRIPTION
**Pull request by gdm85**
The clone button didn't work for me, I managed to copy the missing part from the `new` template.

Also some other minor fix.

### CODE REVIEW

File: templates/paste.html [hunk 1]
```diff
@@ -49,4 +49,28 @@
 
 </form>
 </div>
+
+<div>
+<form class="submit-form" method="post" action="/paste/create.go">
```
Comment (jyap808) [inline templates/paste.html]: Can you modify the action URL?

http.HandleFunc("/paste/create", createHandler)

**Comment by gdm85** [comment]
@jyap808 ops..typo. Fixed now.

**Comment by jyap808** [comment]
Thanks @gdm85!

**Comment by gdm85** [comment]
@jyap808 you're welcome! :) Many thanks to you for the porting effort!

**Pull request merged**